lightningd/plugin.c

```diff
@@ -229,12 +229,11 @@ plugin_request_new_(struct plugin *plugin,
 	    (arg))
 
 /**
- * Given a request, send it to the plugin.
+ * Send a JSON-RPC message (request or notification) to the plugin.
  */
-static void plugin_request_queue(struct plugin *plugin,
-				 struct plugin_request *req)
+static void plugin_send(struct plugin *plugin, struct json_stream *stream TAKES)
```

Comment:
TAKES isn't quite right here: that means it takes ownership iff caller calls take().  Perhaps call it something more obviously "takes-ownership"-is, like "plugin_attach_stream()"?